packages/bot-web-ui/src/stores/dashboard-store.ts

```diff
@@ -222,6 +226,11 @@ export default class DashboardStore implements IDashboardStore {
     filtered_tab_list = [];
     is_chart_modal_visible = false;
     is_trading_view_modal_visible = false;
+    set_faq_title = '';
```

Comment:
@farabi-deriv Can we change the name to `faq_title` 
set implies that it is a method or doer instead of it holding some value.
```suggestion
    faq_title = '';
```

Reply:
@sandeep-deriv yes you're right. Updated the naming. Thanks